db/migrate/20180823203142_change_enum_name.rb

```diff
@@ -0,0 +1,21 @@
+class ChangeEnumName < ActiveRecord::Migration
+  def new_enumlabels(enum_type, old_value, new_value)
```

Comment:
I would still call this `change`

Reply:
Did these migrations work?  You should be using `commercial` instead of `private`